python/hail/representation/genomeref.py

```diff
@@ -151,7 +165,9 @@ def par(self):
         """
 
         from hail.representation.interval import Interval
-        if not self._par:
+        if self._par_jrep is None:
+            self._par_jrep = self._jrep.par()
```

Comment:
is this the only place _par_jrep is used? if so, no need to bind an instance variable for it.